Update to Nerdpack permissions doc (#1230)
newrelic/developer-website: src/markdown-pages/build-apps/permission-manage-apps.mdx

---

There are several restrictions around who can publish, use, and subscribe to Nerdpacks (the file packages that represent [New Relic One applications](https://developer.newrelic.com/build-apps)). 

## Restrictions for basic users

The most important permissions factor is [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#user-type): 

> * **Basic users** can build their own Nerdpacks locally but can't publish them or subscribe accounts to them. They can view apps and visualizations in the [apps catalog](https://opensource.newrelic.com/nerdpacks/) but can't use most of them. 

Reviewer: This reads a little odd, what can I use as a basic user?  Can we specify that?

> * **Full users** can use any Nerdpacks the account they're in has been subscribed to, whether built by New Relic or by others. Full users theoretically have Nerdpack management permissions, but there may be restrictions related to custom role assignments (see below). 

Reviewer: ```suggestion
* **Full users** can use any subscribed Nerdpacks specific to the account they  are associated with, whether built by New Relic or by others. Full users theoretically have Nerdpack management permissions, but there may be restrictions related to custom role assignments (see below). 
```

Reviewer 2: Another suggestion:

> * **Full users** can use any Nerdpacks their accounts have subscribed to. Full users theoretically have Nerdpack management permissions, but there may be restrictions related to custom role assignments (see below).
